### ISSUE TITLE
`tfcth` calculated inconsistently 

### ISSUE THREAD

**Issue by mkovari** [ukaea/PROCESS#3063]
In [PROCESS](https://github.com/ukaea/PROCESS/tree/a4220251bff3ffeab70f5546d83d140b86494c60)/[examples](https://github.com/ukaea/PROCESS/tree/a4220251bff3ffeab70f5546d83d140b86494c60/examples)/csv_output_large_tokamak_MFILE.DAT, `tcfth` is output twice.  The first output is wrong - it shows the initial value, not the final value after iteration.
```
tfcth____________________________________________________ (itvar032)____________________      1.2000E+00

TF_coil_inboard_leg_(m)__________________________________ (tfcth)_______________________      8.6482E-01 ITV
```

This probably results from incomplete implementation of Issue #515.  Note that in this run `thkcas` is *also* an iteration variable:
```
Inboard_leg_case_inboard_"nose"_thickness_(m)____________ (thkcas)______________________      2.2965E-01 ITV
```
In #515 I recommended that we should remove the option of having `tfcth` as an iteration variable, but this obviously hasn't been done.  I recommended that we should use `thkcas` and `thkwp` as iteration variables instead.  There are of course many ways to solve this problem, but over-determining `tfcth` is not one of them.

This is another high priority issue since it could make PROCESS behave erratically and might prevent convergence.
@jonmaddock @jmorris-uk @timothy-nunn @ajpearcey

**Comment by mkovari** [comment]
Tim, If you are going to look at this I would suggest we get some comments from @ajpearcey , @jmorris-uk and @stuartmuldrew.

**Comment by cjwgriesel** [comment]
I am looking at the magnet dimensions as this is inter related with the cable pack problem.

**Comment by mkovari** [comment]
Yes, there is a connection between the overall dimensions of the TF coil cross-section (winding pack and case) and the dimensions of the conductor inside it.  

However, I suggest trying to keep these topics separate as far as humanly possible.

**Comment by mkovari** [comment]
I am going to add a check to stop PROCESS if `tfcth` (inboard TF coil thickness, (centrepost for ST) (m)) is selected as an iteration variable when the TF coil is superconducting (as proposed in Issue #515).
